modules/swagger-codegen/src/main/resources/kotlin-client/data_class.mustache

```diff
@@ -1,3 +1,5 @@
+{{#hasEnums}}import com.squareup.moshi.Json{{/hasEnums}}
+
```

Comment:
Can you remove this extra newline?

Also, I think the newline generated by the hasEnums check when that is false can be eliminated by putting the opening and closing tags on their own lines.

Reply:
@jimschubert Thank you for reviewing, I will remove it soon.